src/core/memory.cc

```diff
@@ -2286,8 +2286,6 @@ void* realloc(void* ptr, size_t size) {
         abort();
     }
     // if we're here, it's a non-null seastar memory ptr
-    // or original functions aren't available.
```

Comment:
This part of the comment was obsolete after some changes to the logic above which means we don't reach here in the "original functions not available" case.